RadarSDK/Radar.h

```diff
@@ -416,6 +424,34 @@ typedef void(^ _Nonnull RadarIPGeocodeCompletionHandler)(RadarStatus status, Rad
 + (void)ipGeocodeWithCompletionHandler:(RadarIPGeocodeCompletionHandler)completionHandler
     NS_SWIFT_NAME(ipGeocode(completionHandler:));
 
+/**
+ TODO(coryp): description and docs
+ Get context for a user's location once in the foreground, without including any user data.
+ 
+ @warning Note that these calls are subject to rate limits.
+ 
+ @param completionHandler An optional completion handler.
+ 
+ @see https://radar.io/documentation/sdk
+ */
++ (void)getContextWithCompletionHandler:(RadarGetContextCompletionHandler _Nullable)completionHandler
```

Comment:
I don't think we necessarily need `WithCompletionHandler` at the end here - this could be just `getContext` that fetches a location fix from the device and sends a context request to our BE

Reply:
hm I might just be misunderstanding as an obj-c rookie, but I think the idea is that the 2 different options that are exposed are for whether you want to use the user's location (prob more common), or if you want to get context for some other location
similar to the 2 trackOnce handlers here: 
https://github.com/radarlabs/radar-sdk-ios/blob/0e09f35c4cd35aec42603f150d6cf716770d277d/RadarSDK/Radar.h#L228-L243

Reply:
sorry, I see how it was ambiguous - this was just a nit about the function name to make it shorter. Theoretically every API call's last argument is the completion handler, so we don't necessarily need to call it out in the function name. Although, it seems like any function called without any arguments follows this convention, so I'm fine with it as is.

Reply:
it would then just be `trackOnce` vs. `trackOnceWithLocation`, anyways..

Reply:
ahhh gotcha, yeah i'll defer to others on the naming convention here :)

Reply:
it's good as is